channels-rke2.yaml

```diff
@@ -1625,7 +1625,19 @@ releases:
   - version: v1.26.14+rke2r1
     minChannelServerVersion: v2.7.5-alpha1
     maxChannelServerVersion: v2.8.99
-    serverArgs: *serverArgs-v1-26-10-rke2r2
+    serverArgs: &serverArgs-v1-26-13-rke2r1
```

Comment:
Version typo on serverArgs but it really doesn't matter in this case so I am fine with just leaving it